pkg/placement/raft/server.go

```diff
@@ -381,6 +381,11 @@ func (s *Server) ApplyCommand(cmdType CommandType, data DaprHostMember) (bool, e
 	s.lock.RLock()
 	defer s.lock.RUnlock()
 
+	// If the command is to upsert a member, but the member has no entities, treat it as a remove command.
+	if cmdType == MemberUpsert && len(data.Entities) == 0 {
+		cmdType = MemberRemove
+	}
```

Comment:
Since our raft store is specialised for storing actor hosts, we don't need to do upsert here, instead a command that says "this host has no actor" is equivalent to removing that host from the table.